graphix/pattern.py

```diff
@@ -66,11 +68,11 @@ def __init__(self, input_nodes=[]):
         self.__Nnode = len(input_nodes)  # total number of nodes in the graph state
         self._pauli_preprocessed = False  # flag for `measure_pauli` preprocessing completion
 
-        self.__seq = []
+        self.__seq: List[command.Command] = []
```

Comment:
If you add `from __future__ import annotations`, you may use `list[command.Command]`.

Reply:
I guess I will also have to echo this change to the other `list` type specifications, along with the tuple ones?